k8s/environments/sbox/common-overlay/neuvector/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../../../namespaces/neuvector
+
+patches:
+- target:
```

Comment:
shouldn't this just be a `patchesStrategicMerge`? https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/cluster-00-overlay/kustomization.yaml#L44

Reply:
modified accordingly